plugins/exporter.koplugin/target/markdown.lua

```diff
@@ -0,0 +1,163 @@
+local _ = require("gettext")
+local UIManager = require("ui/uimanager")
+
+
+local formatters = {
+    none = "%s",
+    bold = "**%s**",
+    italic = "*%s*",
+    bold_italic = "**_%s_**",
+    underline_markdownit = "++%s++",
+    underline_u_tag = "<u>%s</u>",
+    strikethrough = "~~%s~~",
+}
+
+-- markdown exporter
+local MarkdownExporter = require("base"):new {
+    name = "markdown",
+    extension = "md",
+}
+
+
+function MarkdownExporter:loadSettings()
+    local plugin_settings = G_reader_settings:readSetting("exporter") or {}
+    self.settings = plugin_settings[self.name] or {}
+end
+
+
+local formatter_buttons = {
+    {_("None"), "none"},
+    {_("Bold"), "bold"},
+    {_("Bold Italic"), "bold_italic"},
+    {_("Italic"), "italic"},
+    {_("Strikethrough"), "strikethrough"},
+    {_("Underline (Markdownit style, with ++)"), "underline_markdownit"},
+    {_("Underline (with <u></u> tags)"), "underline_u_tag"},
+}
+
+function MarkdownExporter:editFormatStyle(drawer_style)
+    local radio_buttons = {}
+    for _, v in ipairs(formatter_buttons) do
```

Comment:
With `_` here you overwrite `gettext`

Reply:
Yes, but not using `gettext` in the loop.

Reply:
Thank you. FIxed.